core/src/main/java/org/apache/accumulo/core/data/Mutation.java

```diff
@@ -772,6 +772,474 @@ public void putDelete(byte[] columnFamily, byte[] columnQualifier, ColumnVisibil
     put(columnFamily, columnQualifier, columnVisibility.getExpression(), true, timestamp, true, EMPTY_BYTES);
   }
 
+  /**
```

Comment:
The two asterisk at the beginning of the comment indicate that this is javadoc.  However the javadoc block is not associated with a method or class. So I am not sure where this would render when the javadoc html is created.

Could convert it to a regular comment if your intention was that it was just a code comment.  If you want it in the javadoc, maybe move the content to the javadoc for the `at()` method.